API/Controllers/ProjectController.cs

```diff
@@ -226,5 +229,33 @@ public async Task<IActionResult> DeleteProject(int projectId)
             projectService.Save();
             return Ok();
         }
+
+
+        /// <summary>
```

Comment:
I think that this endpoint should get its own controller. It has nothing much to do with the project resource (in our database). This makes the application more modular which can be very helpful in the long run. For example, if we want to convert this application into multiple separate services.

Opinions? @DigitalExcellence/backend